### ISSUE TITLE
[REQ]: Plans for Foundry V12 Support

### ISSUE THREAD

**Issue by FearNot-TheWrath** [Halftonex/npc-generator-gpt#18]
### Description

Curious to know if there is any plans on updating this for Foundry V12.

**Comment by halftonex** [comment]
Yes, definitely. I'm really busy right now but i will do it for sure some time in the future.

**Comment by DiosisNoctum** [comment]
Will you also be updating it to use chatgpt4o?

**Comment by halftonex** [comment]
> Will you also be updating it to use chatgpt4o?

Hey there, sorry for the late reply—it’s been a busy time. Over the next few months, I’m planning to develop a library that will make it easy for developers to integrate LLMs into their modules. As part of that, I’ll be rewriting this module to use the new library and ensure it’s compatible with more game systems. Really appreciate the interest you’ve shown in NPC Generator in the meantime!
